tracing/src/main/java/com/palantir/tracing/Tracer.java

```diff
@@ -138,12 +141,42 @@ public static void initTrace(Optional<Boolean> isObservable, String traceId) {
                 .map(observable -> observable ? Observability.SAMPLE : Observability.DO_NOT_SAMPLE)
```

Comment:
Can't leave a comment above, but we should update the javadoc so that we don't recommend people use a deprecated endpoint

Reply:
linked list style migration docs!